coalib/bears/Bear.py

```diff
@@ -113,8 +113,8 @@ class Bear(Printer, LogPrinter):
     REQUIREMENTS = ()
     AUTHORS = set()
     AUTHORS_EMAILS = set()
-    MAINTAINERS = set()
-    MAINTAINERS_EMAILS = set()
+    MAINTAINERS = AUTHORS
+    MAINTAINERS_EMAILS = AUTHORS_EMAILS
```

Comment:
im not sure, thats why i did not merge it yet, i asked on gitter and you said its enough to do this :p
so should i make it a class property or what?